### ISSUE TITLE
T Separated ISO 8601: Incorrectly Parsed

### ISSUE THREAD

**Issue by piercefreeman** [wanasit/chrono#130]
I'm trying to parse an ISO formatted string like the following one:

`2016-05-07T23:45:00.487+01:00`

This seems like a strict format to me, but when I parse with chrono.strict, it seems to parse out the year as the date itself.

```
ParsedComponents {
  knownValues: { hour: 20, minute: 16, meridiem: 1 },
  impliedValues: { second: 0, millisecond: 0, day: 27, month: 7, year: 2016 } }
```

**Comment by penne12** [comment]
Looks like a fairly simple fix. I don't have much time to work on it right now, but you could send a pr or @wanasit could take a look. 

Honestly, though, if you are only parsing ISO formatted dates, use the native Date.parse(). While it doesn't support natural language, it seems like it does what you want it to do. It's faster, and it's primary focus is to match standards, while Chrono's is to match dates in free-form text. This is still a bug that needs to be taken care of, though.

**Comment by wanasit** [comment]
Fixed in 8037ad1.
